hkmc2/shared/src/test/mlscript/codegen/SanityChecks.mls

```diff
@@ -97,10 +97,14 @@ f(2)
 
 :expect NaN
 :ssjs
+:e
 :noSanityCheck
 data class Cls(x, y) with
   fun f(z, p) = x + y + z + p
 Cls(1, 2).f(3)
```

Comment:
To keep the original intent of the test file:
```suggestion
id(Cls(1, 2)).f(3)
```
(same for the cases below)

Reply:
Applied; and I just realized that I introduced the same problem for this test file in one of my previous PRs; fixed them as well.